pkg/issuer/acme/dns/route53/route53.go

```diff
@@ -145,7 +146,15 @@ func (r *DNSProvider) changeRecord(action, fqdn, value string, ttl int) error {
 
 	resp, err := r.client.ChangeResourceRecordSets(reqParams)
 	if err != nil {
+		if awserr, ok := err.(awserr.Error); ok {
+			if action == route53.ChangeActionDelete && awserr.Code() == route53.ErrCodeInvalidChangeBatch {
+				// If we try to delete something and get a 'InvalidChangeBatch' that
+				// means it's already deleted, no need to consider it an error.
```

Comment:
Looking at the AWS docs, `ErrCodeInvalidChangeBatch` sounds more like a wrapper type for multiple errors from the AWS API? I think this is still okay given the limited set of operations we perform/errors that could occur, but might it be worth logging the returned error somewhere just in case we get some other weird error responses here and someone needs to open an issue/debug?

Reply:
I don't think InvalidChangeBatch will be possible for anything else when the transaction is just a single delete (as is the case here), but no harm logging it.

Reply:
I guess the likes of 503 errors etc are handled with different error response types.

Sounds good to me. I'll lgtm and approve after you've added the log line, if you think it is useful 🙂